utilities/randomPhrases.js

```diff
@@ -7,10 +7,10 @@ const phrases = {
     "It's that time of evening again!",
     ":wave: Whatup peeps?",
     "Drumroll please...:drum_with_drumsticks:",
```

Comment:
This one also needs a space before the emoji, sorry @bryan-johns

Reply:
uff da